backend/src/main/java/de/neuefische/app/spotify/SpotifyApiController.java

```diff
@@ -46,4 +46,15 @@ public SpotifyApiController(PlaylistCSVService playlistCSVService, SpotifyApiSer
         return spotifyApiService.searchPlaylists(value).stream().map(playlist -> PlaylistDTO.of(playlist)).toList();
     }
 
+    @GetMapping
+    public ResponseEntity reloadSpotifyPlaylists(Principal principal){
```

Comment:
`ResponseEntity` is generic. You should define a type parameter.

Reply:
Ok, but can I use it without a Type Parameter, when I just want to return the status? Or should I then use ResponseEntity<Void>? In this case I would use ResponseEntity<String> because I have a body when an exception is thrown

Reply:
Yes if you just want to return the status without body, it is `ResponseEntity<Void>`. And if you don't need a response body and it should always be the same status, your method can return `void` and just needs to be annotated with `@ResponseStatus(HttpStatus...)`.

Reply:
Thanks for explanation